src/components/MerchantsPage/MerchantCard.tsx

```diff
@@ -1,9 +1,23 @@
 import * as React from 'react';
 import { Link } from 'react-router-dom';
-import styles from './styles.module.scss';
 import { BrowsePageSeller } from '../../utilities/api';
-import temp from './temp.png';
 import ProgressBar from '../ProgressBar';
+import styled from 'styled-components';
+import styles from './styles.module.scss';
+
+const Location = styled.div`
+  font-weight: bolder;
+  color: #46accc;
+`;
+
+const Summary = styled.p`
+  display: -webkit-box;
+  height: 45px;
+  -webkit-line-clamp: 2;
+  -webkit-box-orient: vertical;
+  overflow: hidden;
+  text-overflow: ellipsis;
```

Comment:
Could we not use a scss macro for overflow: hidden and text-overflow: ellipsis?

Reply:
if we do that ^ it's only for one line of text which looked kinda weird. 

https://css-tricks.com/almanac/properties/l/line-clamp/